### PR TITLE
WT-11481 Alphabetize Workgen variables

### PR DESCRIPTION
**Pull request by adelinexchen**
Alphabetised variable and function names, excluding some that are already sorted in a specific order i.e. Track struct has getters sorting by duration of time

### CODE REVIEW

File: bench/workgen/workgen.h [hunk 1]
```diff
@@ -341,9 +341,9 @@ struct Operation {
 // properties are prevented, only existing properties can be set.
 struct ThreadOptions {
     std::string name;
-    std::string session_config;
     double throttle;
     double throttle_burst;
+    std::string session_config;
```
Comment (Sean04) [inline bench/workgen/workgen.h:346]: The data types should be grouped together by size so `session_config` should be under `name`

File: bench/workgen/workgen.h [hunk 2]
```diff
@@ -48,9 +48,9 @@ struct OptionsList {
     OptionsList(const OptionsList &other);
     ~OptionsList() = default;
 
-    void add_int(const std::string& name, int default_value, const std::string& desc);
     void add_bool(const std::string& name, bool default_value, const std::string& desc);
     void add_double(const std::string& name, double default_value, const std::string& desc);
+    void add_int(const std::string& name, int default_value, const std::string& desc);
```
Comment (marcbutler) [inline bench/workgen/workgen.h:53]: This is a method/function declaration. So I don't *think* it should be re-ordered but I'm unclear on the exact policy.

File: bench/workgen/workgen.h [hunk 3]
```diff
@@ -323,15 +323,15 @@ struct Operation {
     bool combinable() const;
     void describe(std::ostream &os) const;
 #ifndef SWIG
-    Operation& operator=(const Operation &other);
-    void init_internal(OperationInternal *other);
     void create_all();
-    void get_static_counts(Stats &stats, int multiplier);
+    void init_internal(OperationInternal *other);
```
Comment (marcbutler) [inline bench/workgen/workgen.h:327]: This and several following lines are methods.

File: bench/workgen/workgen.h [hunk 4]
```diff
@@ -437,34 +437,34 @@ struct Transaction {
...
-    int warmup;
-    double oldest_timestamp_lag;
-    double stable_timestamp_lag;
-    double timestamp_advance;
-    bool max_idle_table_cycle_fatal;
-    /* Dynamic create/drop options */
+    int background_compact;
     int create_count;
     int create_interval;
+    /* Dynamic create/drop options */
```
Comment (marcbutler) [inline bench/workgen/workgen.h:443]: Previously this occurred before `create_(count|interval)`.

File: bench/workgen/workgen.h (hunk 2 above)
Comment (Sean04) [reply bench/workgen/workgen.h:53]: The ticket does say to reorder the functions

File: bench/workgen/workgen.h (hunk 1 above)
Comment (adelinexchen) [reply bench/workgen/workgen.h:346]: Fixed https://github.com/wiredtiger/wiredtiger/pull/9970/commits/aae1e5a63d9af7235108e5bf2350b6c74001c6ce

File: bench/workgen/workgen.h (hunk 4 above)
Comment (adelinexchen) [reply bench/workgen/workgen.h:442]: fixed [Moved a comment in that was in the wrong spot and re-grouped variable…](https://github.com/wiredtiger/wiredtiger/pull/9970/commits/ed13ab8cd5f2bb557aea4ae4908f8d520daab6ba)

**Review by Sean04 — APPROVED** [review]
LGTM!

**Comment by adelinexchen** [comment]
evergreen patch failure, unrelated

File: bench/workgen/workgen.h (hunk 2 above)
Comment (Sean04) [reply bench/workgen/workgen.h:53]: @marcbutler

Comment (marcbutler) [reply bench/workgen/workgen.h:53]: Apologies must have missed it.

**Review by marcbutler — APPROVED** [review]
LGTM

**Comment by adelinexchen** [comment]
evergreen merge

**Pull request merged**